common/src/main/java/net/william278/husktowns/visualizer/ParticleChunk.java

```diff
@@ -29,8 +29,13 @@ public class ParticleChunk {
 
     private final Position[] positions = new Position[4];
 
+    Chunk chunk;
+    World world;
+
     private ParticleChunk(@NotNull Chunk chunk, @NotNull World world) {
         super();
+        this.chunk = chunk;
+        this.world = world;
```

Comment:
Why is this necessary?

Reply:
Im need it for create PARTICLE in chunk by location